proposals/v3/api-websocket.md

```diff
@@ -359,14 +360,16 @@ If no timestamp is specified, there will not be an starting/ending bound.
 {
    "id": 1,
    "event": "request/write_async",
-   "data": [
-      {
-         "device": "34c226b1afadaae5f172a4e1763fd1a6",
-         "action": "color",
-         "data": "ff00ff",
-         "transaction": "56a32eba-1aa6-4868-84ee-fe01af8b2e6d"
-      }
-   ]
+   "data": {
+      "id": "34c226b1afadaae5f172a4e1763fd1a6",
+      "data": [
+         {
+            "action": "color",
+            "data": "ff00ff",
+            "transaction": "56a32eba-1aa6-4868-84ee-fe01af8b2e6d"
+         }
+      ]
+   }
```

Comment:
+1 -- this is cleaner and better reflects how the HTTP API operates.

my only suggestion would be to rename the inner `data` to something else (maybe `payload`?) just so we don't have something like `response['data']['data']` which is fine, but not necessarily clear if someone is reading through the code

Reply:
Oh yeah totally. I think `payload` is a good one.